nextcord/errors.py

```diff
@@ -332,9 +326,15 @@ def __init__(self, e: Exception) -> None:
 
 
 class ApplicationCheckFailure(ApplicationError):
-    """Exception raised when the predicates in :attr:`.ApplicationCommand.checks` have failed.
+    """Exception raised when the predicates in :attr:`.BaseApplicationCommand.checks` have failed.
 
     This inherits from :exc:`ApplicationError`
     """
 
     pass
+
+
+class ApplicationCommandOptionMissing(ApplicationError):
```

Comment:
Add this to the docs + exception hierachy

Reply:
DONE.